hooks/path_info.py

```diff
@@ -706,6 +706,55 @@ def copy_files(self, src_files, dest_path, seal_files=False, is_sequence=False):
 
         return processed_files
 
+    def copy_folder(self, src_folders, dest_folder, seal_folder=False):
+        """
+        This method handles copying an item's folder(s) to a designated location.
+
+        """
+
+        publisher = self.parent
+
+        logger = publisher.logger
+
+        # ---- copy the src folders to the dest location
+        processed_folders = []
+        for src_folder in src_folders:
+
+            if not os.path.isdir(src_folder):
+                continue
+
+            # If the folder paths are the same, lock permissions
+            if src_folder == dest_folder:
+                filesystem.freeze_permissions(dest_folder)
+                continue
+
+            # copy the folder method
+            try:
+                filesystem.ensure_folder_exists(dest_folder)
```

Comment:
This is not required.
This is used in `copy_files()` as we are trying to copy a file and the directory it will be put in needs to exist. But in this case, you are trying to make sure the directory destination  path exists, which is unnecessary. It will be created by `filesystem.copy_folder()`